homeassistant/components/transmission/services.yaml

```diff
@@ -9,6 +9,12 @@ add_torrent:
       example: http://releases.ubuntu.com/19.04/ubuntu-19.04-desktop-amd64.iso.torrent
       selector:
         text:
+    download_path:
+      required: false
+      default: None
```

Comment:
I don't think `None` is a good default as it could give the idea that `None` is a valid value for this field

Reply:
Thanks for the remark. Isn't this a standard thing to do? I mean if you fill in `None`, it will ignore the field. Everyone could have their own default locations for download paths, so I find it difficult to come up with a better alternative.

Reply:
I think just empty